lib/core/base/audit.js

```diff
@@ -651,7 +657,8 @@ function getHelpUrl({ brand, application }, ruleId, version) {
 		'/' +
 		ruleId +
 		'?application=' +
-		application
+		application +
+		(lang && lang !== 'en' ? '&lang=' + lang : '')
```

Comment:
Can you escape `lang` here, and maybe while you're at it, do the same for `application`?